backend/httpclient/http_client.go

```diff
@@ -107,8 +107,8 @@ func GetTLSConfig(opts ...Options) (*tls.Config, error) {
 
 	tlsOpts := clientOpts.TLS
 
-	// #nosec
 	config := &tls.Config{
+		// #nosec
```

Comment:
^^^ this is the only change that is not replacing a function name with "_"

I think the security lint rules got more specific -- it was complaining that the value could be true